### ISSUE TITLE
Clicking 'Add [custom-tag]' in dropdown doesn't work

### ISSUE THREAD

**Issue by kmturley** [indrimuska/angular-selector#23]
Where we added the new custom tag stuff it doesn't allowed you to click the custom tag to add it:

```
'<li class="selector-option active" ng-if="filteredOptions.length == 0">' +
    'Add <i ng-bind="search"></i>' +
'</li>' +
```

Suggest you modify to be:

```
'<li class="selector-option active" ng-if="filteredOptions.length == 0" ng-mouseover="highlight(index)" ng-click="addCustom(search)">' +
    'Add <i ng-bind="search"></i>' +
'</li>' +
```

And add the function to scope:

```
scope.addCustom = function (val) {
    if (val) {
        var option = {};
        if (typeof attrs.create === 'function') {
            option = scope.create({ input: val });
        } else {
            option[scope.labelAttr] = val;
            option[scope.valueAttr || 'value'] = val;
        }
        scope.options.push(option);
        scope.set(option);
    }
};
```

Which can be reused here too:

```
case KEYS.enter:
case KEYS.comma:
    if (scope.isOpen) {
        if (scope.filteredOptions.length) {
            scope.set();
        } else if (attrs.create) {
            scope.addCustom(e.target.value);
        }
        e.preventDefault();
    }
    break;
```

**Comment by indrimuska** [comment]
Hey @kmturley, another nice contribution!
Thank you for this, give me 5 to add this feature.

**Comment by indrimuska** [comment]
Now available in [v1.1.2](https://github.com/indrimuska/angular-selector/releases/tag/1.1.2).

**Comment by kmturley** [comment]
nice!!
